src/components/Review/PostCard.tsx

```diff
@@ -0,0 +1,54 @@
+import { css, Theme } from "@emotion/react";
+import { Link } from "@nextui-org/react";
+import { PostMarkdown } from "util/getMarkdownBySlug";
+
+export interface PostListItemProps extends PostMarkdown {}
+
+function PostCard(props) {
+  const post = props;
+  return (
+    <article css={cardWrapper}>
+      <div css={contentWrapper}>
+        <Link href={`/Review/${post?.slug}`}>
```

Comment:
```suggestion
        <Link href={`/Review/${post.slug}`}>
```

slug가 있어야만 동작하는게 맞지 않을까요? 

> 없는 경우가 있다면 따로 Link 태그가 아니도록 핸들링을 하면 될 거 같아요!